pylearn2/training_algorithms/learning_rule.py

```diff
@@ -258,3 +259,83 @@ def get_updates(self, learning_rate, grads, lr_scalers=None):
 
         return updates
 
+
```

Comment:
Looks like it was a long time ago that the Travis test passed and this file has since been de-whitelisted.